packages/data/src/world/oot/overworld.yml

```diff
@@ -1957,8 +1957,8 @@
 "Zora Domain Shortcut":
   region: ZORA_DOMAIN
   exits:
-    "Zora Domain Underwater": "true"
-    "Lake Hylia": "true"
+    "Zora Domain Underwater": "is_adult || can_swim"
```

Comment:
Upon thinking, this exit can be removed. The only reason this sub-region exists is so that you can be safely underwater when coming out of Lake Hylia. There is no purpose in it if you are already in the domain.